src/site/content/en/lighthouse-performance/third-party-facades/index.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: Lazy load third-party resources with facades
+description: |
+  Learn about the opportunities to lazy load third-party resources with facades.
+date: 2020-12-01
+updated: 2020-12-01
+web_lighthouse:
+  - third-party-facades
+---
+
+
+[Third-party resources](/third-party-javascript/) are often used for displaying ads or videos and integrating with social media. The default approach is to load third-party resources as soon as the page loads, but this can unnecessarily slow the page load. If the third-party content is not critical, this performance cost can be reduced by [lazy loading](/fast/#lazy-load-images-and-video) it.
+
+This audit highlights third-party embeds which can be lazily loaded on interaction. In that case, a *facade* is used in place of the third-party content until the user interacts with it. 
+
+{% Aside 'key-term' %}
+
+A *facade* is a static element which looks similar to the actual embedded third-party, but is not functional and therefore much less taxing on the page load.
+
+{% endAside %}
```

Comment:
Great use of the key term aside. When I saw the fancy `facade` word my exact thought was "our readers may not know what that word means"